DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1405,22 +1406,60 @@ public DMASTProcStatementLabel Label(DMASTIdentifier expression)
             List<DMASTDefinitionParameter> parameters = new();
             DMASTDefinitionParameter parameter = DefinitionParameter();
 
-            if (parameter != null || Check(TokenType.DM_IndeterminateArgs)) {
+            if (parameter == null && !Check(TokenType.DM_IndeterminateArgs) && Current().Type != TokenType.DM_RightParenthesis && Current().Type != TokenType.EndOfFile)
+            {
+                // Breaking change - BYOND doesn't specify the arg
+                Error($"error: bad argument definition '{Current().PrintableText}'", false);
+                Recover();
+
+            }
+            else {
                 if (parameter != null) parameters.Add(parameter);
 
+                BracketWhitespace();
+                if (Current().Type != TokenType.DM_RightParenthesis && Current().Type != TokenType.DM_Comma && !Check(TokenType.DM_IndeterminateArgs))
```

Comment:
`DefinitionParameters()` is intended to only parse the vars (`var/a, var/b, var/c`) and not the parentheses. This check (and others) should be moved to `ProcCall()`.